### DISCUSSION TITLE
Paypal webhook don't send notifications to my listener

### DISCUSSION THREAD

**Discussion by makasmi2018** [symfony/symfony#47700] [Q&A]
hello , i have configured webhook url in paypal sandbox mode to test event notificaitons for recurring payments. The problem is that i didn't received any notification in my listener (php symfony controller) ! Thanks for your help in advance

**Comment by Josemm2000** [comment]
Hello, I don't know if you have already solved your problem?, I have a test server and a production server, in my case the production server has been running for more than two weeks, the listener does not send emails (I use swiftmailer). But on the test server it does send them.
